pkg/services/store/validate.go

```diff
@@ -51,11 +47,11 @@ func fail(reason string) validationResult {
 }
 
 func (s *standardStorageService) detectMimeType(ctx context.Context, user *user.SignedInUser, uploadRequest *UploadRequest) string {
-	if strings.HasSuffix(uploadRequest.Path, ".svg") {
-		if svg.IsSVG(uploadRequest.Contents) {
-			return "image/svg+xml"
-		}
-	}
+	// if strings.HasSuffix(uploadRequest.Path, ".svg") {
```

Comment:
it would be helpful in the future to have a code comment above this explaining why it's commented out

Reply:
even better... i kept the IsSVG util :)